docs/tutorial/mac-app-store-submission-guide.md

```diff
@@ -53,7 +52,7 @@ First you need to prepare two entitlements files.
 </plist>
 ```
 
-And then sign your app with following script:
+...and then sign your app with the following script:
```

Comment:
> ...and

I think the "And" is ok here since the previous sentence ends. Also "and" used on line `100` without the "..."